scripts/aigame2.cpp

```diff
@@ -946,7 +946,7 @@ void aiBlackStormTrooper::OnInterval(tOSIRISEventInfo *data) {
         int roll = (rand() % 100);
         if (roll <= 50) {
           find_targets(data->me_handle, false);
-          memory->timer = Game_GetTime() + (memory->target_handle == OBJECT_HANDLE_NONE) ? 5.0f : 2.0f;
+          memory->timer = (Game_GetTime() + (memory->target_handle == OBJECT_HANDLE_NONE)) ? 5.0f : 2.0f;
```

Comment:
Here unclear what meat to be. I'd think that should be

```c
memory->timer = Game_GetTime() + ((memory->target_handle == OBJECT_HANDLE_NONE) ? 5.0f : 2.0f)
```

Reply:
`?` has lower priority than +, that's what clang told me when it triggered the warning. My change is equivalent to the previous version

Reply:
Sentence `(Game_GetTime() + (memory->target_handle == OBJECT_HANDLE_NONE))` is always true as `Game_GetTime()` always positive. So `(T + X) ? 5.0f : 2.0f` is pointless. There intended to be `T + (X ? 5.0f : 2.0f)`

Reply:
Right, it has to be an oversight from the original code